analyses/methylation-summary/tools/03-methyl-tpm-correlation.cwl

```diff
@@ -0,0 +1,48 @@
+cwlVersion: v1.2
+class: CommandLineTool
+id: 03-methyl-tpm-correlation
+doc: >-
+  Calculate representative probe-level correlations between rna-seq (tpm) and methylation (beta/m-vlaues) for patients who have samples in both datasets
+
+requirements:
+  - class: ShellCommandRequirement
+  - class: DockerRequirement
+    dockerPull: "pgc-images.sbgenomics.com/d3b-bixu/open-pedcan:latest"
+  - class: InlineJavascriptRequirement
+  - class: ResourceRequirement
+    ramMin: ${return inputs.ram * 1000}
+    coresMin: 4
+  - class: LoadListingRequirement
+    loadListing: shallow_listing
+  - class: InitialWorkDirRequirement
+    listing:
+      - entryname: 03-methyl-tpm-correlation.py
+        entry:
+          $include: ../03-methyl-tpm-correlation.py
+
+baseCommand: [mkdir]
+
+arguments:
+  - position: 1
+    shellQuote: false
+    valueFrom: |-
+      ./.git
+      python3 03-methyl-tpm-correlation.py
+
+inputs:
+  methyl_values: {type: ['null', {type: enum, name: methyl_values, symbols: ["beta", "m"]}], default: "beta", inputBinding: {prefix: --methyl_values, position: 2}, doc: "OPenPedCan methly matrix values: beta (default) and m"}
+  expression_values: {type: ['null', {type: enum, name: expression_values, symbols: ["gene", "isoform"]}], default: "gene", inputBinding: {prefix: --exp_values, position: 2}, doc: "OpenPedCan expression matrix values: gene (default) and isoform"}
+  histologies: {type: 'File', inputBinding: {position: 4}, doc: "Histologies file"}
+  independent_specimens_rna: {type: 'File', inputBinding: {position: 5}, doc: "OPenPedCan rnaseq independent biospecimen list file"}
```

Comment:
Might be best to match the inputs in cwl named 1-1 with args here: https://github.com/PediatricOpenTargets/OpenPedCan-analysis/blob/81946cce32ce204d78440826e60e771bcda9e03d/analyses/methylation-summary/03-methyl-tpm-correlation.py#L31-L41